expyfun/_utils.py

```diff
@@ -37,6 +37,14 @@
     pass
 
 try:
+    from pyglet.media import avbin
+    del avbin
+except ImportError:
+    has_avbin = False
+else:
+    has_avbin = True
```

Comment:
I know this bit is annoying, but I've seen imports fail for all sorts of reasons. Even doing `except Exception` above wouldn't necessarily take care of all cases, since Pyglet could do something catastrophic with memory. Even import problems aside, there is still import speed to consider. If importing Pyglet or its sub-libraries (including avbin) somehow becomes slow, now importing `expyfun` becomes slow, too.